js/search/searchConfig.js

```diff
@@ -58,7 +58,9 @@ const searchConfig = {
     DISK_NOT_READY: 'NOT_READY',
     DISK_NOT_FOUND: 'DISK_NOT_FOUND',
     MINIMUM_DISK_SPACE: 300000000, // in bytes
-    MAC_PATH_ERROR: "No such file or directory"
+    MAC_PATH_ERROR: "No such file or directory",
```

Comment:
Can you please talk to @VishwasShashidhar and localize this? We can avoid doing this in a separate ticket.